src/psyclone/transformations.py

```diff
@@ -563,98 +562,232 @@ def apply(self, node, options=None):
         return rval
 
 
-class OMPLoopTrans(ParallelLoopTrans):
+class OMPTargetTrans(RegionTrans):
+    '''
+    Adds an OpenMP target directive to a region of code.
+
+    For example:
+
+    >>> from psyclone.psyir.frontend.fortran import FortranReader
+    >>> from psyclone.psyir.nodes import Loop
+    >>> from psyclone.transformations import OMPTargetTrans
+    >>>
+    >>> tree = FortranReader().psyir_from_source("""
+    >>>     subroutine my_subroutine()
```

Comment:
I think these need to be continuation lines (so replace '>>>' with '...') in order for this to be valid Python.

Reply:
Done